democracylab/templates/emails/html_email_frame.html

```diff
@@ -121,8 +121,7 @@
 			padding-bottom: 20px;
 			color: #999999;
 			font-family: sans-serif;" class="footer">
-
-                Copyright &copy; 2019 DemocracyLab. All Rights Reserved.
+<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work by <a xmlns:cc="http://creativecommons.org/ns#" href="https://www.democracylab.org/" property="cc:attributionName" rel="cc:attributionURL">https://www.democracylab.org/</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>
```

Comment:
Can you break this up into multiple lines for easier readability?

Reply:
@marlonkeating Im on it right now.